content/zh/docs/queue_resource_management.md

```diff
@@ -0,0 +1,144 @@
++++
+title = "队列资源管理"
+
+date = 2024-12-30
+lastmod = 2024-12-30
+
+draft = false  # Is this a draft? true/false
+toc = true  # Show table of contents? true/false
+type = "docs"  # Do not modify.
+
+# Add menu entry to sidebar.
+[menu.docs]
+  parent = "features"
+  weight = 1
++++
+
+## 功能概述
+
+[队列](/zh/docs/queue)是Volcano的核心概念之一，用于支持多租户场景下的资源分配与任务调度。通过队列，用户可以实现多租资源分配、任务优先级控制、资源抢占与回收等功能，显著提升集群的资源利用率和任务调度效率。
+
+### 核心特性
+
+1. 灵活的资源配置
+
+   * 支持多维度资源配额控制(CPU、内存、GPU、NPU等)
+   * 提供三级资源配置机制:
+      * Capability: 队列资源使用上限
+      * Deserved: 资源应得量（在无其他队列提交作业时，该队列内作业所占资源量可超过Deserved值，当有多个队列提交作业且集群资源不够用时，超过Deserved值的资源量可以被其他队列回收）
+      * Guarantee: 资源预留量（预留资源只可被该队列所使用，其他队列无法使用）
+
+      > 建议: 
+      > 1. 进行三级资源配置时，需遵循: Guarantee <= Deserved <= Capability；
+      > 2. Guarantee可按需配置，在开启Capacity插件时需要配置Deserved值；
+      > 3. Deserved配置建议：在平级队列场景，所有队列的Deserved值总和等于集群资源总量；在层级队列场景，子队列的Deserved值总和等于父队列的Deserved值。
+
+   * 支持动态资源配额调整
+
+
+2.[层级队列](/zh/docs/hierarchical_queue)管理
```

Comment:
1 and 2 are not aligned.

Reply:
updated

![image](https://github.com/user-attachments/assets/7a7cf378-8e78-450b-8785-fc26aa62136c)